gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.3-bin.zip
```

Comment:
question: Is this something we want to be updating?

Reply:
No, not in this PR. It should be updated by the upstream maintainers or as a separate PR (one topic/one PR). 

I needed to update or downgrade my java version; I chose to update, but will revert before PR w/ upstream.